org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/opentelemetry/PeriodicAsyncMeasurement.java

```diff
@@ -118,6 +122,10 @@ boolean requiresAsyncMeasurement() {
     return true;
   }
 
+  protected boolean isExecuteAsyncMeasurementOnThisNode() {
```

Comment:
Perhaps the name of this method as well as the supplier that provides the return value could be phrased more generally, such as `isActiveOnThisNode`.